demo/src/main/java/com/eis/geoCalendar/demo/Behaviour/wrapper/EventMapBehaviourWrapper.java

```diff
@@ -0,0 +1,181 @@
+package com.eis.geoCalendar.demo.Behaviour.wrapper;
+
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.fragment.app.FragmentManager;
+
+import com.eis.geoCalendar.demo.Behaviour.EventMapBehaviour;
+import com.eis.geoCalendar.demo.Behaviour.LocationRetriever;
+import com.eis.geoCalendar.demo.Behaviour.MapBehaviour;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventCreatedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventRemovedListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnEventTriggeredListener;
+import com.eis.geoCalendar.demo.Behaviour.listener.OnMapInitializedListener;
+import com.eis.geoCalendar.demo.Bottomsheet.BottomSheetBehaviour;
+import com.eis.geoCalendar.demo.Dialogs.AbstractAddEventDialog;
+import com.eis.geoCalendar.demo.Dialogs.AbstractRemoveEventDialog;
+import com.eis.geoCalendar.demo.Localization.GoToGoogleMapsNavigator;
+import com.eis.geoCalendar.events.Event;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.List;
+
+/**
+ * Class that wraps public methods for {@link com.eis.geoCalendar.demo.Behaviour.EventMapBehaviour}
+ * following interface {@link com.eis.geoCalendar.demo.Behaviour.MapBehaviour}
+ * <p>
+ * If needed, the class {@link com.eis.geoCalendar.demo.Behaviour.EventMapBehaviour} can be set
+ * to package-private and this wrapper can be used outside of the package
+ *
+ * @author Turcato
+ */
+public class EventMapBehaviourWrapper implements MapBehaviour<Event<String>> {
```

Comment:
If you have time, I think you should put a {@link className} on specs of all methods of this class, when they refer to another class/interface.